cmd/subcmds/acl.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package subcmds
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/np-guard/vpc-network-config-synthesis/pkg/ir"
+	"github.com/np-guard/vpc-network-config-synthesis/pkg/synth"
+)
+
+func NewACLCommand(args *inArgs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "acl",
+		Short: "Generate  Networks ACLs from connectivity specification",
+		Long: `Generate Network ACLs to only allow the specified connectivity, either for each subnet separately or per VPC.
+		Endpoints in the required-connectivity specification may be subnets, subnet segments, CIDR segments and externals.`,
+		RunE: func(_ *cobra.Command, _ []string) error {
```

Comment:
```suggestion
		Long: `Generate Network ACLs to only allow the specified connectivity, either for each subnet separately or per VPC.
		Endpoints in the required-connectivity specification may be subnets, subnet segments, CIDR segments and externals.`,
		Args: cobra.NoArgs,
		RunE: func(_ *cobra.Command, _ []string) error {
```
To prevent running something like `./bin/vpcgen acl blabla -c test/data/acl_testing5/config_object.json`